src/components/ChatLog.jsx

```diff
@@ -0,0 +1,46 @@
+import ChatEntry from './ChatEntry';
+import PropTypes from 'prop-types';
+
+
+const ChatLog = (props) => {
+  const chatentrycomponents = props.entries.map((entries, index) => {
```

Comment:
We could maybe name this `entry` instead of `entries` since it only a single `entry` we will be looking at each iteration.